python/utils.py

```diff
@@ -677,6 +677,12 @@ def fcnRegexp(pattern, string):
             break
     if not found:
         raise RuntimeError("Cannot find any suitable spatialite module")
+    if any(['.gpkg' in arg for arg in args]):
+        try:
+            cur.execute("SELECT EnableGpkgAmphibiousMode()")
+        except Exception as e:
+            print(e)
+            raise Exception("Could not enable geopackage amphibious mode")
```

Comment:
- Does it make sense to catch and re-raise here? What's the rationale behind this?
- Don't `print` the exception. The python console is not a place to report errors. If that needs to be logged, use a builtin error reporting mechanism. But since this is in `utils.py` we don't really know if it's going to be used in desktop / standalone / server, so maybe it's the wrong place anyway for any kind of logging / reporting and instead that should be done on a different level and here we only raise an exception (or let the exception go through).

Reply:
The issue is that enabling this function is nice and can be useful, people might even depend on it for some things but on the other hand I don't want to break the connection when this cannot be enabled. Yet I want the user to know when this could not be enabled. 
Since this might be used as a standalone or in the app I'm not sure how to make a call that will be properly handled.

Reply:
I'll just settle for a warn.